### ISSUE TITLE
Unformatted reference

### ISSUE THREAD

**Issue by haji-ali** [inukshuk/jekyll-scholar#303]
(Apologies for my many questions)

Is there a way to get an unformated `{{reference}}`? Devoid of html tags and html escapes characters? I don't want to remove the formatting completely, I want to have the two versions of the reference (one with and one without).

**Comment by inukshuk** [comment]
I don't think so. It's would be easy to add this if you need it. You could add a second [renderer instance](https://github.com/inukshuk/jekyll-scholar/blob/master/lib/jekyll/scholar/utilities.rb#L668) with `text` format instead of `html`.

**Comment by inukshuk** [comment]
Alternatively, you could use the `render_sort` method of the renderer. This will return a slightly different version of the reference which is used for bibliography sorting internally; but it would probably be what you need here.

**Comment by haji-ali** [comment]
Is that doable in the config? Or do you mean by modifying `utilities.rb`?

I temporarily solved the issue with the `strip_html` filter, but I do think this would be a good functionality to have (I use the plain text version for a copy button).

I might have a crack at it at some point.

**Comment by inukshuk** [comment]
You'd have to modify jekyll-scholar, it's not possible to do this via the config unless I'm missing something.

The functionality to render plain text instead of HTML is definitely implemented by the citeproc renderer though: I think the easiest approach would be as outlined above to create a second renderer using the text output format, and render that output and then [expose it along with the other reference data to the page](https://github.com/inukshuk/jekyll-scholar/blob/master/lib/jekyll/scholar/utilities.rb#L541)
